services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/current_subscription.jsx

```diff
@@ -44,7 +44,7 @@ export default class CurrentSubscription extends React.Component {
       return <span>Credit Card</span>;
     } else if (subscriptionType === 'School Sponsored' || subscriptionType === 'Trial') {
       return <span>No Payment Method on File</span>;
-    } else if (subscriptionStatus && (!subscriptionStatus.payment_method || subscriptionStatus.payment_method === 'School Invoice')) {
+    } else if (subscriptionStatus && ['Invoice', 'School Invoice'].includes(subscriptionStatus.payment_method)) {
```

Comment:
Just confirming that 'Invoice' on its own isn't a valid subscription type? Otherwise we presumably do still want to show that here, right?